docs/NON_DEVCONTAINER_SETUP.md

```diff
@@ -1,54 +1,80 @@
 [Back to *Chat with your data* README](../README.md)
 
-# Project setup
+# Non-DevContainer Setup
 
-If you are unable to run this accelerator as a DevContainer or in CodeSpaces, then you will need.
+If you are unable to run this accelerator using a DevContainer or in GitHub CodeSpaces, then you will need to install the following prerequisites on your local machine.
 
-- [Visual Studio Code](https://code.visualstudio.com/)
-    - Extensions
-        - [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
-        - [Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
-        - [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
-        - [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)
-        - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-        - [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) (optional: Teams extension only)
-- Install [Node.js](https://nodejs.org/en)
-  - Install the LTS version (Recommended for Most Users)
+- A code editor. We recommend [Visual Studio Code](https://code.visualstudio.com/), with the following extensions:
+  - [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
+  - [Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+  - [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
+  - [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)
+  - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+  - [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) **Optional**
+- [Python 3.11](https://www.python.org/downloads/release/python-3119/)
+- [Node.js LTS](https://nodejs.org/en)
+- [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd)
+- [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local)
 
+## Setup
 
-### Running the sample using the Azure Developer CLI (azd)
+1. Configure your Python environment:
```

Comment:
Could they not just run the postcreate script themselves?

Reply:
Technically, yes, they could. However, I am usually against asking users to run a script and would rather document the steps required. This makes it easier for folk to understand the process and increases transparency (particularly for those who would run it without checking what it does).

Reply:
I agree and disagree on this one as this one means we have to keep the documentation up to date rather than asking them: - 

Please run the following script after you have checked the contents
```bash
./.devcontainer/postCreate.sh
```

We could rename the script to be `setupDependencies.sh` to be more agnostic